CHANGELOG.md

```diff
@@ -4,36 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [0.8.0]
```

Comment:
@JakeUrban can you add a link to be able to see the difference between both versions? Something similar to (https://github.com/stellar/ruby-stellar-base/compare/v0.21.0...v0.22.0) (check out stellar-base)